zmq-async/src/zmq_async.ml

```diff
@@ -1,89 +1 @@
-open Base
-open Async_kernel
-open Async_unix
-open Sexplib.Conv
-
-module Socket = struct
-  exception Break_event_loop [@@deriving sexp_of]
```

Comment:
I have removed the sexp stuff, as AFAICS it is not used anywhere in the zmq code. Removing it allows us to not depend on base for the zmq-lwt version